docs/data/base/components/select/select.md

```diff
@@ -0,0 +1,93 @@
+---
+product: base
+title: React Select unstyled component and hook
+components: SelectUnstyled, MultiSelectUnstyled, OptionUnstyled, OptionGroupUnstyled
+githubLabel: 'component: select'
+waiAria: https://www.w3.org/TR/wai-aria-practices/#combobox
+packageName: '@mui/base'
+---
+
+# Select
+
+<p class="description">The SelectUnstyled and MultiSelectUnstyled components let you create menus of options for users to choose from.</p>
+
+{{"component": "modules/components/ComponentLinkHeader.js", "design": false}}
+
+MUI Base offers two components to replace the native HTML `<select>` tag: `SelectUnstyled` and `MultiSelectUnstyled`.
+
+## SelectUnstyled
+
+```tsx
+import SelectUnstyled from '@mui/base/SelectUnstyled';
+```
+
+### Basic usage
+
+{{"demo": "UnstyledSelectSimple.js", "defaultCodeOpen": false}}
+
+The `SelectUnstyled` component accepts generic props.
+Due to TypeScript limitations, this may cause unexpected behavior when wrapping the component in `forwardRef` (or other higher-order components).
+In such cases, the generic argument will be defaulted to `unknown` and type suggestions will be incomplete.
+To avoid this, you can manually cast the resulting component to the correct type, as shown in the demo above.
```

Comment:
How about we show a short code snippet of this instead of asking developers to open the previous demo?